project_task_programmed/models/project_task_alert.py

```diff
@@ -57,15 +55,12 @@ def _merge_placeholders(self, incoming, rec):
     def create_task_alerts(self):
         for task_alert in self:
             days_delta = timedelta(days=task_alert.days_delta)
-            to_date = (datetime.now() + days_delta).strftime(DF)
+            to_date = fields.Datetime.now() + days_delta
```

Comment:
Careful, this won't work: `fields.Datetime.now() ` returns a string in the server expected format.
It´s perfect for a field default value, but not for date math.